creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -57,7 +58,15 @@ const healthCheckController = async (req) => {
   let { randomBytesToSign } = req.query
 
   const logger = req.logger
-  const response = await healthCheck(serviceRegistry, logger, sequelize, getMonitors, numberOfCPUs, randomBytesToSign)
+  const response = await healthCheck(
+    serviceRegistry,
+    logger,
+    sequelize,
+    getMonitors,
+    numberOfCPUs,
+    TranscodingQueue.getTranscodeQueueJobs,
+    randomBytesToSign
```

Comment:
this needs to be in verbose check instead right

Reply:
i added it to both checks, see below for how verbose calls into this

Reply:
oh yeah my local state just had this one - cool, don't think we should have it in the base health check tbh

Reply:
i can remove from regular health check for now but longer term i think we should consider just combine these health checks

Reply:
yeah i know - down to discuss that (i made a jam session note since i'm not convinced) but yeah for now just minimal changes to ship this would be best

Reply:
alright removed from regular health check